pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -102,10 +110,9 @@ static int _rename(vfs_mount_t *mountp, const char *from_path,
     char fatfs_abs_path_to[FATFS_MAX_ABS_PATH_SIZE];
     fatfs_desc_t *fs_desc = (fatfs_desc_t *)mountp->private_data;
 
-    snprintf(fs_desc->abs_path_str_buff, FATFS_MAX_ABS_PATH_SIZE, "%d:/%s",
-             fs_desc->vol_idx, from_path);
+    _build_abs_path(fs_desc, from_path);
 
-    snprintf(fatfs_abs_path_to, sizeof(fatfs_abs_path_to), "%d:/%s",
+    snprintf(fatfs_abs_path_to, sizeof(fatfs_abs_path_to), "%u:/%s",
```

Comment:
This format string is duplicated with `_build_abs_path`. Would it make sense to put the format string in a shared define to remove the `magic` value in the middle of the functions?

Reply:
I guess it could be macroed, but I don't think it is necessary with only two occurrences. Before this PR, the format string was duplicated in all functions.

Reply:
Ok